### DISCUSSION TITLE
changing image location breaks Frigate

### DISCUSSION THREAD

**Discussion by sh00tfire** [blakeblackshear/frigate#7380] [General]
Hi,

Recently noticed that there is a new image path.  I have been running this in my docker-compose.  
image: blakeblackshear/frigate:stable
When I change to this, I get errors starting docker-compose
image: ghcr.io/blakeblackshear/frigate:stable

not sure what I am seeing in the log as it starts but here is what I see.  

frigate    | s6-rc: info: service s6rc-fdholder: starting
frigate    | s6-rc: info: service s6rc-oneshot-runner: starting
frigate    | s6-rc: info: service s6rc-oneshot-runner successfully started
frigate    | s6-rc: info: service fix-attrs: starting
frigate    | s6-rc: info: service s6rc-fdholder successfully started
frigate    | s6-rc: info: service fix-attrs successfully started
frigate    | s6-rc: info: service legacy-cont-init: starting
frigate    | s6-rc: info: service legacy-cont-init successfully started
frigate    | s6-rc: info: service log-prepare: starting
frigate    | s6-rc: info: service log-prepare successfully started
frigate    | s6-rc: info: service nginx-log: starting
frigate    | s6-rc: info: service go2rtc-log: starting
frigate    | s6-rc: info: service frigate-log: starting
frigate    | s6-rc: info: service nginx-log successfully started
frigate    | s6-rc: info: service go2rtc-log successfully started
frigate    | s6-rc: info: service go2rtc: starting
frigate    | s6-rc: info: service frigate-log successfully started
frigate    | s6-rc: info: service go2rtc successfully started
frigate    | s6-rc: info: service go2rtc-healthcheck: starting
frigate    | s6-rc: info: service frigate: starting
frigate    | s6-rc: info: service go2rtc-healthcheck successfully started
frigate    | s6-rc: info: service frigate successfully started
frigate    | s6-rc: info: service nginx: starting
frigate    | 2023-08-03 14:56:49.953373638  [INFO] Preparing go2rtc config...
frigate    | s6-rc: info: service nginx successfully started
frigate    | s6-rc: info: service legacy-services: starting
frigate    | 2023-08-03 14:56:49.956419419  [INFO] Starting Frigate...
frigate    | 2023-08-03 14:56:49.958571589  [INFO] Starting NGINX...
frigate    | s6-rc: info: service legacy-services successfully started
frigate    | 2023-08-03 14:56:50.147819486  [INFO] Starting go2rtc...
frigate    | 2023-08-03 14:56:50.281323935  14:56:50.281 INF go2rtc version 1.2.0 linux/amd64
frigate    | 2023-08-03 14:56:50.281761867  14:56:50.281 INF [api] listen addr=:1984
frigate    | 2023-08-03 14:56:50.282030319  14:56:50.282 INF [rtsp] listen addr=:8554
frigate    | 2023-08-03 14:56:50.282279466  14:56:50.282 INF [srtp] listen addr=:8443
frigate    | 2023-08-03 14:56:50.282517988  14:56:50.282 INF [webrtc] listen addr=:8555
frigate    | 2023-08-03 14:56:50.453109057  2023/08/03 14:56:50 [error] 135#135: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.29.8, server: , request: "GET /api/stats HTTP/1.1", upstream: "http://127.0.0.1:5001/stats", host: "hp-linux.wrightplanet.local:5000"
frigate    | 2023-08-03 14:56:50.453136111  192.168.29.8 - - [03/Aug/2023:14:56:50 +0000] "GET /api/stats HTTP/1.1" 502 157 "-" "HomeAssistant/2023.8.0 aiohttp/3.8.5 Python/3.11" "-"
frigate    | 2023-08-03 14:56:51.048851154  [2023-08-03 14:56:51] frigate.app                    INFO    : Starting Frigate (0.12.0-da3e197)
frigate    | 2023-08-03 14:56:51.064023841  *************************************************************
frigate    | 2023-08-03 14:56:51.064027873  *************************************************************
frigate    | 2023-08-03 14:56:51.064031775  ***    Your config file is not valid!                     ***
frigate    | 2023-08-03 14:56:51.064034014  ***    Please check the docs at                           ***
frigate    | 2023-08-03 14:56:51.064035481  ***    https://docs.frigate.video/configuration/index     ***
frigate    | 2023-08-03 14:56:51.064049562  *************************************************************
frigate    | 2023-08-03 14:56:51.064050915  *************************************************************
frigate    | 2023-08-03 14:56:51.064052686  ***    Config Validation Errors                           ***
frigate    | 2023-08-03 14:56:51.064054091  *************************************************************
frigate    | 2023-08-03 14:56:51.064055328  Camera bedroom_cam has rtmp enabled, but rtmp is not assigned to an input.
frigate    | 2023-08-03 14:56:51.064881854  Traceback (most recent call last):
frigate    | 2023-08-03 14:56:51.064884571    File "/opt/frigate/frigate/app.py", line 351, in start
frigate    | 2023-08-03 14:56:51.064885990      self.init_config()
frigate    | 2023-08-03 14:56:51.064887519    File "/opt/frigate/frigate/app.py", line 81, in init_config
frigate    | 2023-08-03 14:56:51.064888812      self.config = user_config.runtime_config
frigate    | 2023-08-03 14:56:51.064890189    File "/opt/frigate/frigate/config.py", line 1002, in runtime_config
frigate    | 2023-08-03 14:56:51.064891414      verify_config_roles(camera_config)
frigate    | 2023-08-03 14:56:51.064892798    File "/opt/frigate/frigate/config.py", line 770, in verify_config_roles
frigate    | 2023-08-03 14:56:51.064893981      raise ValueError(
frigate    | 2023-08-03 14:56:51.064895395  ValueError: Camera bedroom_cam has rtmp enabled, but rtmp is not assigned to an input.
frigate    | 2023-08-03 14:56:51.064896515
frigate    | 2023-08-03 14:56:51.064897854  *************************************************************
frigate    | 2023-08-03 14:56:51.064911504  ***    End Config Validation Errors                       ***
frigate    | 2023-08-03 14:56:51.064912802  *************************************************************
frigate    | 2023-08-03 14:56:52.224865309  [INFO] Service Frigate exited with code 1 (by signal 0)
frigate    | s6-rc: info: service legacy-services: stopping
frigate    | s6-rc: info: service legacy-services successfully stopped
frigate    | s6-rc: info: service nginx: stopping
frigate    | s6-rc: info: service go2rtc-healthcheck: stopping
frigate    | 2023-08-03 14:56:52.234281915  [INFO] The go2rtc-healthcheck service exited with code 256 (by signal 15)
frigate    | s6-rc: info: service go2rtc-healthcheck successfully stopped
frigate    | 2023-08-03 14:56:52.311847710  [INFO] Service NGINX exited with code 0 (by signal 0)
frigate    | s6-rc: info: service nginx successfully stopped
frigate    | s6-rc: info: service nginx-log: stopping
frigate    | s6-rc: info: service frigate: stopping
frigate    | s6-rc: info: service frigate successfully stopped
frigate    | s6-rc: info: service go2rtc: stopping
frigate    | s6-rc: info: service frigate-log: stopping
frigate    | s6-rc: info: service nginx-log successfully stopped
frigate    | 2023-08-03 14:56:52.321462125  exit OK
frigate    | s6-rc: info: service frigate-log successfully stopped
frigate    | 2023-08-03 14:56:52.334448918  [INFO] The go2rtc service exited with code 0 (by signal 0)
frigate    | s6-rc: info: service go2rtc successfully stopped
frigate    | s6-rc: info: service go2rtc-log: stopping
frigate    | s6-rc: info: service go2rtc-log successfully stopped
frigate    | s6-rc: info: service log-prepare: stopping
frigate    | s6-rc: info: service s6rc-fdholder: stopping
frigate    | s6-rc: info: service log-prepare successfully stopped
frigate    | s6-rc: info: service legacy-cont-init: stopping
frigate    | s6-rc: info: service s6rc-fdholder successfully stopped
frigate    | s6-rc: info: service legacy-cont-init successfully stopped
frigate    | s6-rc: info: service fix-attrs: stopping
frigate    | s6-rc: info: service fix-attrs successfully stopped
frigate    | s6-rc: info: service s6rc-oneshot-runner: stopping
frigate    | s6-rc: info: service s6rc-oneshot-runner successfully stopped

My docker-compose.yml

version: "3.9"
services:
  frigate:
    container_name: frigate
    privileged: true # this may not be necessary for all setups
    restart: unless-stopped
    #image: blakeblackshear/frigate:stable
    #image: ghcr.io/blakeblackshear/frigate:0.12.0
    image: ghcr.io/blakeblackshear/frigate:stable
    shm_size: "128mb" # update for your cameras based on calculation above
    devices:
      - /dev/bus/usb:/dev/bus/usb # passes the USB Coral, needs to be modified for other versions
      - /dev/apex_0:/dev/apex_0 # passes a PCIe Coral, follow driver instructions here https://coral.ai/docs/m2/get-started/#2a-on-linux
      - /dev/dri/renderD128 # for intel hwaccel, needs to be updated for your hardware
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - /etc/localtime:/etc/localtime:ro
      - /home/travis/frigate/config.yml:/config/config.yml:ro
      - /home/travis/frigate/storage:/media/frigate
      - type: tmpfs # Optional: 1GB of memory, reduces SSD/SD Card wear
        target: /tmp/cache
        tmpfs:
          size: 1000000000
    ports:
      - "5000:5000"
      - "1935:1935" # RTMP feeds
      - "8554:8554" # Birdseye
    environment:
      FRIGATE_RTSP_PASSWORD: "password"

my config.yml

mqtt:
  host: 192.168.29.8
  user: mqtt_user
  password: pwd

birdseye:
  enabled: True
  mode: continuous

cameras:
  bedroom_cam:
    ffmpeg:
      inputs:
        - path: rtsp://rtspadmin:Pass@wyzecam01.wrightplanet.local/live
          roles:
            - detect
            - clips
            - record
    objects:
      track:
        - car
        - person

    snapshots:
      enabled: True
      timestamp: False
      bounding_box: True
      retain:
        default: 14
    record:
      enabled: True
      events:
        retain:
          default: 7
        pre_capture: 5
        post_capture: 8
       
        
    rtmp:
      enabled: True # <-- RTMP should be disabled if your stream is not H264

    detect:
      width: 1920
      height: 1080
      fps: 20
    motion:
      mask:
        - 1920,1000,1920,1080,1377,1080,1372,1004
        - 0,469,1920,469,1920,0,0,0
  
  
  
  Front_Porch_Cam:
    ffmpeg:
      inputs:
        - path: rtsp://rtspadmin:Pass@10.1.29.4/live
          roles:
            - detect
            - clips
            - record  
    objects:
      track:
        - car
        - person
        - cat
        - dog
        
    snapshots:
      enabled: True
      timestamp: False
      bounding_box: True
      retain:
        default: 14
    record:
      enabled: True
      events:
        retain:
          default: 7
        pre_capture: 5
        post_capture: 8
    detect:
      width: 1920
      height: 1080
      fps: 20
        
    motion:
      mask:
        - 1872,1000,1387,1004,1382,1080,1875,1080
        - 753,0,304,0,592,725,917,667
        - 1219,0,1920,0,1920,218
detectors:
  cpu1:
    type: cpu
  cpu2:
    type: cpu
    
environment_vars:
  PLUS_API_KEY: REDACTED

**Comment by NickM-27** [comment]
You get errors because you are pulling a newer version of frigate. I'd suggest reading the 0.12.0 release notes

**Reply by sh00tfire** [reply]
didn't realize it was so different.  thanks for the info

**Comment by blakeblackshear** [comment]
You posted your plus api key. Please generate a new one.

**Reply by sh00tfire** [reply]
done.  thanks
